### DISCUSSION TITLE
Improve usability of moving claim corners

### DISCUSSION THREAD

**Discussion by hippotastic** [GriefPrevention/GriefPrevention#1306] [Ideas]
I see users on my server struggling with moving claim corners almost on a daily basis.

Here are their common mistakes:
1. Left-clicking instead of right-clicking on the highlighted corner markers with the golden shovel to start resizing a corner.
2. Not clicking on the glowstone block to start resizing a claim, but on the gold blocks right next to it.
3. Left-clicking instead of right-clicking with the golden shovel to set the new corner location.
4. Standing right next to their existing claim instead of inside their existing claim when equipping the golden shovel, and thereby not triggering the automatic highlighting of the existing claim.

I think that the following improvements could be made to avoid all of these common issues:
1. Intercept left-clicks with the golden shovel on claim corners and start the resize operation instead of breaking the clicked block.
2. Allow clicks on ANY part of the claim corner highlight to start the resize operation of the clicked corner.
3. While moving a claim corner is in progress, intercept left-clicks with the golden shovel and make them move the corner to the clicked location instead of breaking the clicked block.
4. If the player equips the golden shovel and does NOT stand inside a claim, highlight their adjacent claims.

Thank you for reading and considering this suggestion. It would greatly reduce my admin overhead helping players understand how to resize their claims.

**Comment by RoboMWM** [comment]
I'm all for ideas on improving controls. Commands are one way to deal with it, but either way the whole concept of "visualizing" the claim can be difficult depending on the player's experience and maturity.

the gherkin statement isn't so much to be used as a "feedback" sort of response. I guess in a way it does dictate the motivation yes, but there's no specific feature being asked for here, so the "what" for the motivation makes it fall flat and feel weird, lol.

Oh ok I see some ideas being thrown around in an ordered list. Could be made more visible but yea.

**Reply by hippotastic** [reply]
Ok, I removed the user story part.

I also love the idea of resizing claims in a drag-and-drop fashion without having to rely on commands. That's why I want my users to succeed with their attempts and not struggle with the inputs which seem to feel unintuitive to some.
